tests/unit/test_service_steps.py

```diff
@@ -692,10 +692,30 @@ def test_databrew_start_job_run_step_creation_sync():
 
 
 @patch.object(boto3.session.Session, 'region_name', 'us-east-1')
-def test_databrew_start_job_run_step_creation():
-    step = GlueDataBrewStartJobRunStep('Start Glue DataBrew Job Run', wait_for_completion=False, parameters={
-        "Name": "MyWorkflowJobRun"
-    })
+def test_databrew_start_job_run_step_creation_wait_for_completion():
+    step = GlueDataBrewStartJobRunStep(
+        'Start Glue DataBrew Job Run - WaitForCompletion', integration_pattern=IntegrationPattern.WaitForCompletion,
+        parameters={
+            "Name": "MyWorkflowJobRun"
+        })
```

Comment:
I think there should be a test for verifying the invalid (unsupported) service integration pattern as well. This was not a case that was previously possible but needs test coverage now.

Reply:
Agreed - we should cover this case as well! 
Changes will be added in the next commit

Reply:
Added unit test to cover unsupported integration pattern in last commit